DataFormats/L1TParticleFlow/interface/PFCluster.h

```diff
@@ -0,0 +1,76 @@
+#ifndef DataFormats_L1TParticleFlow_PFCluster_h
+#define DataFormats_L1TParticleFlow_PFCluster_h
+
+#include <vector>
+#include "DataFormats/L1Trigger/interface/L1Candidate.h"
+#include "DataFormats/Common/interface/Ref.h"
+
+namespace l1t {
+
+  class PFCluster : public L1Candidate {
+  public:
+    /// constituent information. note that this is not going to be available in the hardware!
+    typedef std::pair<edm::Ptr<l1t::L1Candidate>, float> ConstituentAndFraction;
+    typedef std::vector<ConstituentAndFraction> ConstituentsAndFractions;
+
+    PFCluster() {}
+    PFCluster(float pt,
```

Comment:
this constructor could call the other constructor (to reduce duplication)

Reply:
These are two different constructors, on with PolarLorentzVector one with LorentzVector. So leaving as is.